a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -421,13 +421,13 @@ <h4>Variables setup</h4>
                        </span>
                     </li>
 
-                    <li><b>LET</b> <var>certifier</var> be the value of the node extracted from <var>package_document</var>, using the xpath <code class="xpath">/package/metadata/meta[@property="a11y:<i>certifiedBy</i>"]/text()</code>.</li>
+                    <li><b>LET</b> <var>certifier</var> be the value of the node extracted from <var>package_document</var>, using the xpath <code class="xpath">/package/metadata/meta[@property="a11y:<i>certifiedBy</i>"]/normalize-space()</code>.</li>
```

Comment:
These node extraction calls need to have normalize-space(.) to be valid. I think there are a few text() uses in the onix doc, as well, but not as many as the epub one. I'll push changes for these in a few minutes.